spec/requests/pages_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe PagesController, type: :request do
+  let(:policy) { GetIntoTeachingApiClient::PrivacyPolicy.new(id: "123", text: "Latest privacy policy") }
```

Comment:
I'd suggest replacing `123` with `policy_id` and swap the lines round - it'll be easier to maintain in the future

Reply:
Good idea

Reply:
I think... the factory is built within the test suit, so may use the files in the AcceptPrivacyPolicy cassettes, would have to investigate